package/base-files/files/sbin/sysupgrade

```diff
@@ -213,10 +213,14 @@ build_list_of_backup_overlay_files() {
 	( cd /overlay/upper/; find .$SAVE_OVERLAY_PATH \( -type f -o -type l \) $find_filter | sed \
 		-e 's,^\.,,' \
 		-e '\,^/etc/board.json$,d' \
+		-e '\,^/etc/luci-uploads/.placeholder,d' \
 		-e '\,/[^/]*-opkg$,d' \
 		-e '\,^/etc/urandom.seed$,d' \
+		-e '\,^/etc/apk/world$,d' \
```

Comment:
@aparcar any clue if it's ok to always wipe world file?

Reply:
The normal backup is already ignoring that file. I'm just wiping it from "-o".